src/app/components/footer/footer.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input } from '@angular/core';
+import { FooterService } from 'src/app/core/services/footer.service';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss']
+})
+
+export class FooterComponent {
+
+  @Input() copyrightYear: number = 2020;
```

Comment:
We can describe this directly in template, or in constants, @Input is not neccessery